packages/ui/atomic/create-atomic-component-project/template/src/components/results-manager/templates/youtube-template.html

```diff
@@ -0,0 +1,71 @@
+<style>
+  .field {
+    display: inline-flex;
+    white-space: nowrap;
+    align-items: center;
+  }
+
+  .field-label {
+    font-weight: bold;
+    margin-right: 0.25rem;
+  }
+
+  .smooth-border {
+    border-radius: 5px;
+    overflow: hidden;
+  }
+
+  .force-fit-content {
+    height: fit-content !important;
```

Comment:
We really !important ?
I have ptcssd when i see this keyword xD

Reply:
I know. me too. Other option would be to use a more specific selector to overwrite the height property defined in atomic:
`.result-root.with-sections.display-list.image-small atomic-result-section-visual.force-fit-content`

Reply:
https://github.com/coveo/ui-kit/blob/639d19c57940607d85e431e1a042ee941c3cf6c7/packages/atomic/src/pages/examples/fashion.html#L171-L209

I think `atomic-result-section-badges.force-fit-content` might work

Reply:
If a component needs to manipulate the height or positioning of a OOTB section, that means one of two things:
1. The original behaviour is a bug, we should fix it in Atomic.
2. Our case requires customization that isn't needed for most other result templates. We should not use sections at all.
 
In principle, only Atomic should be responsible for the layout of result sections. If we try to forcefully change part of it, then future changes in Atomic may break the layout. We should strongly discourage this.

In this particular case, what does it look like without `fit-content`? I wonder which of the above two cases it is.

Reply:
Without the fit-content, the image dimension is fixed to 10rem x 10rem 
![image](https://user-images.githubusercontent.com/12199712/220954578-a30c779f-be50-499f-b7ea-290300754dfb.png)
I think this makes sense images with a 1:1 width/height ratio (which is not the case here). Otherwise, the space not used by the image is left blank.

I used `fit-content` to only use the necessary height needed by the image
![image](https://user-images.githubusercontent.com/12199712/220955015-fcb291fb-f2b4-4a57-a874-66f0ad04ec8d.png)

Reply:
@btaillon, does it make sense to have this change directly in Atomic?

Reply:
Is this with `image-size` small?

You can change the image-size on a per-template basis by adding `image-size="large"` to the visual section.

Reply:
The image still has a 1:1 ratio
![image](https://user-images.githubusercontent.com/12199712/221220847-6b8136d7-1047-46dc-973a-737bed18d3e4.png)

Reply:
Part of why I forced a 1:1 ratio was because, on mobile results (or any results where the image is at the top), it ensured the titles were aligned. However, I do see in this case that it doesn't really make sense...

I think for desktop list results in Atomic, it would be fine to add `fit-content` (for everyone using Atomic), or at-least align the contents of the section to the top.

Reply:
Created a JIRA 
https://coveord.atlassian.net/browse/KIT-2314